.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "standard"
-}
\ No newline at end of file
+  "extends": "standard",
+  "rules": {
+    "no-unused-expressions": 1
```

Comment:
https://travis-ci.org/karma-runner/karma-mocha/jobs/350197732#L2134

```
   99:11  warning  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  167:11  warning  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  257:11  warning  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  527:7   warning  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  537:7   warning  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  552:7   warning  Expected an assignment or function call and instead saw an expression  no-unused-expressions
```

for chai APIs